corehq/apps/integration/forms.py

```diff
@@ -101,6 +101,17 @@ class GaenOtpServerSettingsForm(forms.ModelForm):
         label=_("Enable GAEN OTP Server Integration"),
         required=False
     )
+
+    GAEN_SERVER_TYPE_OPTIONS = (('Nearform', 'Nearform OTP Server'), ('APHL', 'APHL Exposure Notifications'),)
+    gaen_server_type = forms.CharField(
+        label=_('GAEN Server Type'),
+        widget=forms.Select(choices=[
```

Comment:
This can reference `GaenOtpServerSettings.GAEN_SERVER_TYPE_OPTIONS`, which could be renamed to something like `SERVER_OPTIONS` since it's in the `GaenOtpServerSettings` model.